target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-deco-m5.dts

```diff
@@ -0,0 +1,361 @@
+// SPDX-License-Identifier: ISC
+// Copyright (c) 2015, The Linux Foundation. All rights reserved.
+// Copyright (c) 2021, Frank Veltmans <veltmans.frank@gmail.com>
+
+ 
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/soc/qcom,tcsr.h>
+
+/ {
+	model = "TP-Link Deco M5";
+	compatible = "tplink,deco-m5", "qcom,ipq4019";
```

Comment:
This looks really similar to v2 DTS, can you make a share DTSI to avoid duplication.
Other than that, same comments from v2 DTS apply here